charts/lumigo-operator/templates/cluster-agent-otel-collector-config.yaml.tpl

```diff
@@ -0,0 +1,121 @@
+{{- define "clusterAgent.otelCollectorConfig" -}}
+receivers:
+  filelog:
+    include:
+      - /var/log/pods/*/*/*.log
+    exclude:
+      - /var/log/pods/kube-system_*/*/*.log
+      - /var/log/pods/lumigo-system_*/*/*.log
+      # Exclude logs from the otlp-sink in tests (Kind cluster)
+      - /var/log/pods/local-path-storage_*/*/*.log
+      - /var/log/pods/otlp-sink_*/*/*.log
+    start_at: end
+    include_file_path: true
+    include_file_name: false
+    operators:
+      # Find out which format is used by kubernetes
+      - type: router
+        id: get-format
+        routes:
+          - output: parser-docker
+            expr: 'body matches "^\\{"'
+          - output: parser-crio
+            expr: 'body matches "^[^ Z]+ "'
+          - output: parser-containerd
+            expr: 'body matches "^[^ Z]+Z"'
+      # Parse CRI-O format
+      - type: regex_parser
+        id: parser-crio
+        regex:
+          '^(?P<time>[^ Z]+) (?P<stream>stdout|stderr) (?P<logtag>[^ ]*) ?(?P<log>.*)$'
+        output: overwrite-log-body
+        timestamp:
+          parse_from: attributes.time
+          layout_type: gotime
+          layout: '2006-01-02T15:04:05.999999999Z07:00'
+      # Parse CRI-Containerd format
+      - type: regex_parser
+        id: parser-containerd
+        regex:
+          '^(?P<time>[^ ^Z]+Z) (?P<stream>stdout|stderr) (?P<logtag>[^ ]*) ?(?P<log>.*)$'
+        output: overwrite-log-body
+        timestamp:
+          parse_from: attributes.time
+          layout: '%Y-%m-%dT%H:%M:%S.%LZ'
+      # Parse Docker format
+      - type: json_parser
+        id: parser-docker
+        output: overwrite-log-body
+        timestamp:
+          parse_from: attributes.time
+          layout: '%Y-%m-%dT%H:%M:%S.%LZ'
+      - type: move
+        id: overwrite-log-body
+        from: attributes["log"]
+        to: body
+      # best effort attempt to parse the body as a stringified JSON object
+      - type: json_parser
+        parse_to: body
+        on_error: send_quiet
```

Comment:
This is the only change in this file, that was moved from being an inline config in the `cluster-agent.yaml` file